boards/mcb2388/doc.txt

```diff
@@ -0,0 +1,58 @@
+/**
+@defgroup    boards_mcb2388 MCB2388 Evaluation Board
+@ingroup     boards
+@brief       Support for the MCB2388 Evaluation Board
+
+## Overview
+
+![MCB2300 image](http://www.keil.com/mcb2300/mcb2300.jpg)
```

Comment:
I'm not sure how friendly the guys at keil are about using their images. If it would be still actively sold, this would actually be kind of an advertisement and they should be fine with this. But maybe it is better to check (or ask) in advance, than finding out they don't like other sides using their pics via a copyright infringement notice from a lawyer.

Reply:
When I grep for *.jpg in `boards/` there is a lot of hot-linking to Microchip, Digikey & ST already.
If it calms your mind I can take a picture myself, but it may not turn out as pretty.

Reply:
If the were no angry letters by lawyers so far, I'd guess it is reasonable to assume they won't pop up with one link more.